test/e2e/specs/wp-invite-users-spec.js

```diff
@@ -382,249 +381,4 @@ describe( `[${ host }] Invites:  (${ screenSize })`, function () {
-			await navbarComponent.dismissGuidedTours();
-			await navbarComponent.clickCreateNewPost();
-
-			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
-			await gEditorComponent.enterTitle( reviewPostTitle );
-			return await gEditorComponent.enterText( postQuote );
-		} );
-
-		step( 'New user can submit the new post for review as pending status', async function () {
-			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
-			await gEditorComponent.submitForReview();
-			await gEditorComponent.ensureSaved();
-			return await gEditorComponent.closeEditor();
-		} );
-
-		step( 'New user can see post on posts page in pending status', async function () {
-			const postsPage = await PostsPage.Expect( driver );
-			await postsPage.viewMyPosts();
-			await postsPage.viewDrafts();
-			await postsPage.waitForPostTitled( reviewPostTitle );
-			const pending = await postsPage.isPostPending();
-			return assert( pending, 'The pending post was not displayed on the posts page' );
-		} );
-
-		step( 'As the original user, can see new user added to site', async function () {
-			await new LoginFlow( driver ).loginAndSelectPeople();
-			const peoplePage = await PeoplePage.Expect( driver );
-			await peoplePage.selectTeam();
-			await peoplePage.searchForUser( newUserName );
-			const numberPeopleShown = await peoplePage.numberSearchResults();
-			return assert.strictEqual(
-				numberPeopleShown,
-				1,
-				`The number of people search results for '${ newUserName }' was incorrect`
-			);
-		} );
-
-		step(
-			'As the original user, I can change the contributor user to an author user',
-			async function () {
-				const peoplePage = await PeoplePage.Expect( driver );
-
-				await peoplePage.selectOnlyPersonDisplayed();
-				const editTeamMemberPage = await EditTeamMemberPage.Expect( driver );
-				await editTeamMemberPage.changeToNewRole( 'author' );
-				const noticesComponent = await NoticesComponent.Expect( driver );
-				const displayed = await noticesComponent.isSuccessNoticeDisplayed();
-				return assert(
-					displayed,
-					'The update successful notice was not shown on the edit team member page.'
-				);
-			}
-		);
-
-		step( 'As the invited user, I can now publish a post', async function () {
-			const loginPage = await LoginPage.Visit( driver );
-			await loginPage.login( newUserName, password );
-			await ReaderPage.Expect( driver );
-			const navBarComponent = await NavBarComponent.Expect( driver );
-			await navBarComponent.clickCreateNewPost();
-
-			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
-			await gEditorComponent.enterTitle( publishPostTitle );
-			await gEditorComponent.enterText( postQuote );
-			await gEditorComponent.ensureSaved();
-			return await gEditorComponent.publish( { visit: true } );
-		} );
-	} );
-
-	// Disabled pending wp-calypso issue 26178
-	describe.skip( 'Inviting New User as a Follower: @parallel @jetpack', function () {
```

Comment:
Disabled in #30616

Reply:
I guess it is disabled due to a valid bug, which isn't fixed yet